src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -691,6 +697,10 @@ public function load(array $configs, ContainerBuilder $container)
             $definition->addTag('messenger.message_handler', $tagAttributes);
         });
 
+        $container->registerAttributeForAutoconfiguration(AsSchedule::class, static function (ChildDefinition $definition, AsSchedule $attribute, \ReflectionClass $reflector): void {
```

Comment:
The last parameter is not necessary, right?

```suggestion
        $container->registerAttributeForAutoconfiguration(AsSchedule::class, static function (ChildDefinition $definition, AsSchedule $attribute): void {
```